src/boot/stylesheets/main.scss

```diff
@@ -39,20 +39,18 @@ body {
     	color: $blue-wordpress;
     }
 
-    a[disabled], .wpnc__single-view header .back[disabled] {
-    	pointer-events: none;
-    	color: $gray;
-    }
-
     button {
 		background-color: transparent;
 		border: none;
+		color: $blue-wordpress;
 		cursor: pointer;
 		font-size: inherit;
+		outline: none;
 		padding: 0;
 
-		&.focus {
-			outline: none;
+		&[disabled] {
+			pointer-events: none;
```

Comment:
It is needed, yes, since without it, it inherits the `cursor: pointer` property of `.wpnc__main button` (which seems a sensible default, being the most common state).